### ISSUE TITLE
Default values not working with simpl-schema module

### ISSUE THREAD

**Issue by Gervwyk** [vazco/uniforms#130]
I've started building a uniforms-antd package to support Ant Design. Struggled to get the defaultValues working (the 'defaultValue' prop is undefined in the components), however after a while of debugging I noticed its also not working on the bootstrap versions. I substituted simpl-schema with aldeed:simple-schema and that solved the problem. 

Not sure what the problem is in simpl-schema but for now I'm reverting back to the meteor package.

**Comment by radekmie** [comment]
What exactly is not working?

Here are few _probably_ related places in the code:
[`BaseField#getFieldProps`](https://github.com/vazco/uniforms/blob/master/packages/uniforms/src/components/fields/BaseField.js#L150-L241)
[`SimpleSchema2Bridge#getInitialValue`](https://github.com/vazco/uniforms/blob/master/packages/uniforms/src/bridges/SimpleSchema2Bridge.js#L97-L115)
[`connectField#componentWillMount`](https://github.com/vazco/uniforms/blob/master/packages/uniforms/src/helpers/connectField.js#L36-L51)

**Comment by Gervwyk** [comment]
To fix the problem I simply replaced:

`import SimpleSchema from 'simpl-schema';`

with:

`import {SimpleSchema} from 'meteor/aldeed:simple-schema';`

in `demo\assets\schemas.js ` and then defaultValue was set correctly in all components. Will give simpl-schema another go later this week to see if its related to my code. I'm running the demo code and only adding a new style package - uniforms-antd
Heres the fork only just started:
https://github.com/Gervwyk/uniforms/tree/uniforms-antd

**Comment by radekmie** [comment]
Just show me your schema. There's probably some misunderstanding or some minor bug in `SimpleSchema2Bridge` implementation.

---

Nice work! How about creating a PR with the current progress of your theme package?

**Comment by Gervwyk** [comment]
```
'Example - address': schema`{
        street: {
            type: String,
            max: 100,
            defaultValue: 'aaaa'
        },
        city: {
            type: String,
            max: 50,
            uniforms: {
                type: 'textarea',
                rows: 3
            }
        },
        zip: {
            type: String,
            regEx: /^[0-9]{5}$/,
            defaultValue: '11111'
        },
        number: {
            type: Number,
            label: "Number Test"
        },
        numberDec: {
            type: Number,
            label: "Number Test Deci",
            decimal: true,
            uniforms: {step: 0.1 }
        },
        boolfield: {
            type: Boolean,
            defaultValue: true
        },
        "selectTest": {
            type: String,
            allowedValues: ['aaa','bbbb','cccc','dddd'],
        },
        "multiselectTest": {
            type: [Object],
            allowedValues: [{value: 'aaa', label: 'a'},{value: 'bbb', label: 'b'},{value: 'ccc', label: 'c'},{value: 'ddd', label: 'd'}],
            uniforms: {multiple: true}
        },
        datesamplefield: {
          type: Date,
          label: "This is a date component",
          defaultValue: new Date(2015,10,2,3,4,5),
          uniforms: {
            showTime: false,
            format: "DD-MM-YYYY",
            placeholder: "Select Date",
            disabled: false,
            allowClear: false
          }
        }
    }`
```

This is what I have thus far. Working with aldeed package. Will switch back later this weekend to investigate. 

Sweet! Will send PR when base components is done hopefully sometime next week. Great work on uniforms! Saved me 100s of hours on current project!

**Comment by radekmie** [comment]
Great! I'll give it a try tomorrow.

**Comment by radekmie** [comment]
OK, I found it. [New SimpleSchema strips `defaultValue`](https://github.com/aldeed/node-simple-schema/blob/master/lib/SimpleSchema.js#L756-L769). I'll try to make a workaround.

**Comment by radekmie** [comment]
I've _hacked_ it a little bit. It's published in `v1.6.0-beta.3`.

**Comment by Gervwyk** [comment]
Thanks! This is great news. Will see if I can switch to new simple schema later this week.
